app/src/components/pages/collectionsPage/collectionsPage.tsx

```diff
@@ -23,11 +22,13 @@ import {
 } from "@/src/config/constants";
 import { SearchManagerFactory } from "@/src/utils/searchManager";
 import { headerBreakpoints } from "@/src/utils/breakpoints";
+import DCSearchBar from "../../search/dcSearchBar";
 
 export function CollectionsPage({ data, collectionSearchParams }) {
   const { push } = useRouter();
   const pathname = usePathname();
   const headingRef = useRef<HTMLHeadingElement>(null);
+  const isFirstLoad = useRef<boolean>(false);
```

Comment:
I gave you this variable name and now that I'm reading it, it could probably be better named. Maybe `isAfterFirstLoad`? Mostly because it's true every time except on the initial load.